### PR TITLE
chore: bump func from 0.18 to 0.21

### PR DESCRIPTION
**Pull request by lance**
# Changes

- :broom: add formulae for versions 0.19, 0.20 and 0.21
- :broom: make func 0.21.2 the latest

/kind enhancement

Fixes: https://github.com/knative-sandbox/kn-plugin-func/issues/794

**Release Note**
```release-note
Adds versions 0.19, 0.20 and 0.21 of func
```

### CODE REVIEW

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

File: func@0.18.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class Func < Formula
```
Comment (rhuss) [inline func@0.18.rb:3]:
```suggestion
class FuncAT018 < Formula
```

You need to rename the ruby class to something unique (see also the other, version specific fomulas)

File: func@0.19.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class Func < Formula
```
Comment (rhuss) [inline func@0.19.rb:3]:
```suggestion
class FuncAT019 < Formula
```

see comment above.

File: func@0.20.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require 'fileutils'
+
+class Func < Formula
```
Comment (rhuss) [inline func@0.20.rb:3]:
```suggestion
class FuncAT020 < Formula
```

**Review by rhuss — CHANGES_REQUESTED** [review]
Looks good except for the class naming. See comments inline.

**Comment by lance** [comment]
@rhuss ty for the feedback - obviously I was unaware! Updated now, ptal.

**Review by dsimansk — APPROVED** [review]
/approve
/lgtm

**Pull request merged**